### PR TITLE
Add and remove leading and trailing slashes to avoid a 301

### LINKED ISSUE #43: Ensure all API URLs have correct trailing slash
By default, Django redirects slashless paths to slashful ones; we should circumvent the redirect by making use of the correct flavor of URLs internally.

### PR DESCRIPTION
**Pull request by JackWilb**
### Does this PR close any open issues?
Closes #43

### Give a longer description of what this PR addresses and why it's needed
I noticed that I was getting 301ed a lot when using multinet and with the UPDB deployment, the 301 was leading to bad gateway. I'm not certain why that would be, but I'm hoping that these changes help fix the problem.

### Provide pictures/videos of the behavior before and after these changes (optional)
N/A

### Are there any additional TODOs before this PR is ready to go?

TODOs:
- [ ] Make a release after merge

### CODE REVIEW

File: src/axios.ts [hunk 1]
```diff
@@ -210,15 +210,15 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
   };
 
   Proto.deleteNetwork = function(workspace: string, network: string): AxiosPromise<string> {
-    return this.delete(`/workspaces/${workspace}/networks/${network}`);
+    return this.delete(`/workspaces/${workspace}/networks/${network}/`);
```
Comment (waxlamp) [inline src/axios.ts:213]: Initial `/`?

File: src/axios.ts [hunk 2]
```diff
@@ -75,15 +75,15 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
   const Proto = Object.getPrototypeOf(axiosInstance);
 
   Proto.logout = function(): void {
-    this.get('/user/logout');
+    this.get('user/logout');
```
Comment (waxlamp) [inline src/axios.ts:78]: Trailing `/`?

File: src/axios.ts [hunk 3]
```diff
@@ -75,15 +75,15 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
...
   };
 
   Proto.userInfo = function(): AxiosPromise<UserSpec | null> {
-    return this.get('/users/me');
+    return this.get('users/me');
```
[the remark this answers is not in the document]
Comment (JackWilb) [reply src/axios.ts:82]: No trailing slash for this. From the django debug 404 page: `api/users/me`

File: src/axios.ts (hunk 2 above)
Comment (JackWilb) [reply src/axios.ts:78]: I actually can't find this route in the API and the url in the code doesn't work at all (trailing slash or not)

File: src/axios.ts [hunk 4]
```diff
@@ -180,11 +180,11 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
   };
 
   Proto.deleteTable = function(workspace: string, table: string): AxiosPromise<string> {
-    return this.delete(`/workspaces/${workspace}/tables/${table}`);
+    return this.delete(`/workspaces/${workspace}/tables/${table}/`);
```
[the remark this answers is not in the document]
Comment (JackWilb) [reply src/axios.ts]: See 42d8778

File: src/axios.ts (hunk 1 above)
Comment (JackWilb) [reply src/axios.ts]: See 42d8778

File: src/axios.ts [hunk 5]
```diff
@@ -210,15 +210,15 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
...
   };
 
   Proto.aql = function(workspace: string, query: string): AxiosPromise<any[]> {
-    return this.get(`/workspaces/${workspace}/aql`, { params: {query} });
+    return this.get(`/workspaces/${workspace}/aql/`, { params: {query} });
   };
 
   Proto.createAQLTable = function(workspace: string, table: string, query: string): AxiosPromise<any[]> {
-    return this.post(`/workspaces/${workspace}/tables`, query, {
+    return this.post(`/workspaces/${workspace}/tables/`, query, {
```
[the remark this answers is not in the document]
Comment (JackWilb) [reply src/axios.ts]: See 42d8778

File: src/axios.ts [hunk 6]
```diff
@@ -233,7 +233,7 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
   };
 
   Proto.uploads = function(workspace: string): AxiosPromise<any> {
-    return this.get(`/workspaces/${workspace}/uploads`);
+    return this.get(`/workspaces/${workspace}/uploads/`);
```
[the remark this answers is not in the document]
Comment (JackWilb) [reply src/axios.ts]: See 42d8778

File: src/axios.ts [hunk 7]
```diff
@@ -210,15 +210,15 @@ export function multinetAxiosInstance(config: AxiosRequestConfig): MultinetAxios
   };
 
   Proto.deleteNetwork = function(workspace: string, network: string): AxiosPromise<string> {
-    return this.delete(`/workspaces/${workspace}/networks/${network}`);
+    return this.delete(`/workspaces/${workspace}/networks/${network}/`);
   };
 
   Proto.aql = function(workspace: string, query: string): AxiosPromise<any[]> {
-    return this.get(`/workspaces/${workspace}/aql`, { params: {query} });
+    return this.get(`/workspaces/${workspace}/aql/`, { params: {query} });
```
[the remark this answers is not in the document]
Comment (JackWilb) [reply src/axios.ts]: See 42d8778

**Pull request merged**